CHANGES.md

```diff
@@ -1,3 +1,31 @@
+# 2.19.0 - 27 Feb 2025
+
+## Features
+
+* Expose the Juju version via Model objects in (#1563)
+* Support starting and stopping Pebble checks, and the checks enabled field in (#1560)
+
+## Fixes
+
+* Expose the mocked Juju environment variables via os.environ again in (#1573)
+/canonical/operator/pull/1573)
```

Comment:
This one is a Scenario only change, so doesn't need to be in the ops CHANGES.md. (And it's [already in the Scenario changes](https://github.com/canonical/operator/blob/main/testing/CHANGES.md) because of the bugfix release we did).